Gemfile

```diff
@@ -13,8 +13,9 @@ group :development, :test do
   gem 'climate_control'
 end
 
-gem 'azure_mgmt_resources', '~>0.5.0'
-gem 'azure_mgmt_compute',   '~>0.5.0'
-gem 'azure_mgmt_network',   '~>0.5.0'
-gem 'azure_mgmt_storage',   '~>0.5.0'
-gem 'haikunator',           '~>1.1'
\ No newline at end of file
+gem 'azure_mgmt_resources', '~>0.6.0'
+gem 'azure_mgmt_compute',   '~>0.6.0'
+gem 'azure_mgmt_network',   '~>0.6.0'
+gem 'azure_mgmt_storage',   '~>0.6.0'
+gem 'haikunator',           '~>1.1'
+
```

Comment:
@sarangan12 With your comments i got very interested in understanding this idea. I found an interesting thing.

GitHub itself trails the last trailing line if the file has it or shows some symbol. Now in your changes GitHub is showing one empty line that implies that you must have two empty line in this file at the end. It'd be good if you can remove extra unnecessary line. Let me know if i am misunderstood. 

Thanks for helping me understand this :)